### PR TITLE
Panama

### PR DESCRIPTION
**Pull request by Vesihiisi**
Mapping: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/pa_(es)
Preview: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/pa_(es)/preview

Task: https://phabricator.wikimedia.org/T176731

### CODE REVIEW

File: importer/PaEs.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        There's a number of 'weird' values
+        like 'Cultural heritage monuments in $province'
+        so detect those and don't add them!
+        """
+        if self.commonscat.startswith("Cultural heritage monuments"):
+            return
+        else:
+            super().set_commonscat()
+
+    def exists_with_monument_article(self, language):
```
Comment (lokal-profil) [inline importer/PaEs.py]: Sometimes also uses wikilinked `nombre`

Comment (lokal-profil) [reply importer/PaEs.py]: Done

File: importer/PaEs.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    def set_commonscat(self):
+        """
+        Set the Commons cat property.
+
+        There's a number of 'weird' values
+        like 'Cultural heritage monuments in $province'
+        so detect those and don't add them!
+        """
+        if self.commonscat.startswith("Cultural heritage monuments"):
+            return
```
[the remark this answers is not in the document]
Comment (lokal-profil) [reply importer/PaEs.py]: Done. See https://gist.github.com/Vesihiisi/f37e4c87624c056aed93eeb0a99353b9 for cases where this kicks in

**Comment by Vesihiisi** [comment]
Updated preview here.

**Comment by lokal-profil** [comment]
Preview looks good to me

**Pull request merged**